k8s_node_selector/README.MD

```diff
@@ -0,0 +1,39 @@
+# K8s Node Selector
+
+This is a simple example of an OPA policy which will apply a `nodeSelector` to each pod seen. 
+
+The purpose is to assign the workloads from certain `namespaces` to different `agentpools`.  
+
+## How to specify which `namespaces` map to which `agentpools`?
+
+Firstly the policy assumes that each agent pool has a label `agentpool: nameHere` assigned to all nodes within it. 
+
+Update the `namespaceToAgentPool` mapping in [`main.rego`](./main.rego) to specify which work ends up where. 
+
+You can also update the `ignoredNamespaces` to ensure namespaces, like `kube-system`, are ignored. 
+
+## Deployment
+
+Use `make docker-e2e` to test locally. This will: 
+
+1. Build a docker image with all the pre-reqs (OPA, KIND, OpenSSL and python)
+2. Create a KIND Kubernetes cluster locally 
+3. Deploy Open Poligy Agent Gateway into the cluster
+4. Deploy `main.rego` into the cluster
+5. Use [the Pyton integration tests](./integration/test/) to validate the labels are applied correctly. 
+
+To develop iteratively I recommend using [VSCode Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) adding which will start a VSCode instance within the [devcontainer](./.devcontainer). 
+This means all dependencies, addings and settings are ready to go. Once the adding is installed open the folder and `CTRL+SHIFT+P` -> `Reopen in container`. 
+
+Once VSCode has built the container you can use: 
+
+- `make kind-start` to create a test cluster
+- `make kind-deploy` to deploy OPA and `main.rego` into the cluster
+- `make integration` to run the integration tests against the cluster
+- `make integration-update-rego` to push an update to `main.rego`
+- `make logs` to get the logs from OPA in the cluster
+
+
```

Comment:
Could you expand somewhere in the README about what's contained in this directory?  Why, for example, is there rego code in `k8s_node_selector` and different Rego code in the `rego` directory?  Are there 2 use cases, or is there just 1?  

Said another way, do both those rego directories get loaded into the same OPA and used for mutating admission control?   If so are they used together or separately?

Reply:
Sorry, I managed to mess up a file copy hence the duplication. Will fix this up.

Reply:
Done, fixed up the commit history and files also added description of the folder structure in the readme.